packages/core/src/browser/pageExitObservable.ts

```diff
@@ -51,6 +51,6 @@ export function createPageExitObservable(configuration: Configuration): Observab
   })
 }
 
-export function isPageExitReason(reason: string | undefined): reason is PageExitReason {
-  return includes(objectValues(PageExitReason), reason)
+export function isPageExitReason(reason: string): reason is PageExitReason {
+  return objectValues(PageExitReason).includes(reason as any)
```

Comment:
```suggestion
  return Object.values(PageExitReason).includes(reason as PageExitReason);
```

Reply:
I kept `objectValues` for now, I plan to tackle them in another PR